com.unity.render-pipelines.high-definition/Runtime/Compositor/CustomClear.cs

```diff
@@ -35,9 +35,7 @@ protected override void Setup(ScriptableRenderContext renderContext, CommandBuff
             // Setup code here
             if (string.IsNullOrEmpty(name)) name = "CustomClear";
 
-            var hdrpAsset = HDRenderPipeline.defaultAsset;
-            if (hdrpAsset != null)
-                m_FullscreenPassMaterial = CoreUtils.CreateEngineMaterial(hdrpAsset.renderPipelineResources.shaders.customClearPS);
+            m_FullscreenPassMaterial = CoreUtils.CreateEngineMaterial(HDRenderPipelineGlobalSettings.instance.renderPipelineResources.shaders.customClearPS);
```

Comment:
Before we checked against null asset. It is not done anymore. The HDRenderPipelineGlobalSettings.instance can still be null?